<ui version="4.0" >
 <class>WdgPanoramaCreation</class>
 <widget class="QWidget" name="WdgPanoramaCreation" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="layoutWidget" >
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Source files:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listImages" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>7</hsizetype>
           <vsizetype>3</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonAddImages" >
           <property name="text" >
            <string>add image</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget" >
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="labelPreview" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>5</hsizetype>
           <vsizetype>7</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Click on the preview button to show a preview of the panorama...</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>141</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonPreview" >
           <property name="text" >
            <string>Preview</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonCreatePanorama" >
           <property name="text" >
            <string>Create Panorama</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonCancel" >
           <property name="text" >
            <string>Cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
